testing/test_many.out

```diff
@@ -2,12 +2,12 @@ Running testing/test_many.sh
 Many Tests
 DAQ stress test
 Enough results: 1
-Enough DHCP timeouts: 1
+Enough DHCP timeouts: 0
```

Comment:
this seems wrong -- why are these changing and/or failing?

Reply:
because there aren't any DHCP timeouts

Reply:
why aren't there any DHCP timeouts?